### PR TITLE
Added additional parameters visibilityTimeout and waitTimeSeconds

### PR DESCRIPTION
**Pull request by apinnecke**
Just needed to override the VisibilityTimeout and waitTimeSeconds. :)

### CODE REVIEW

**Comment by robinjmurphy** [comment]
Hi @apinnecke. I'm not too sure whether we should set defaults. I believe if you leave the values as `undefined`, the AWS SDK will just do the right thing?

**Comment by apinnecke** [comment]
This is not about setting defaults, this is about injecting values into the
constructor. ;)

Besides the 20 sec the default value 0 for the visiblityTimeout is taken
from SQS Docs.
Am 21.10.2015 5:28 nachm. schrieb "Robin Murphy" <notifications@github.com>:

File: index.js [hunk 1]
```diff
@@ -40,6 +42,9 @@ function Consumer(options) {
   this.messageAttributeNames = options.messageAttributeNames || [];
   this.stopped = true;
   this.batchSize = options.batchSize || 1;
+  this.visibilityTimeout = options.visibilityTimeout || 0;
```
Comment (robinjmurphy) [inline index.js]: Just talking about changing this from

```
options.visibilityTimeout || 0;
```

to

```
options.visibilityTimeout;
```

This way we don't need to care what the default value (`0`) is.

Comment (robinjmurphy) [inline index.js]: I think the default for this is actually 30 seconds, which is why I'd rather not include a default value here.

**Comment by apinnecke** [comment]
Okay, fixed that :)

**Comment by robinjmurphy** [comment]
Will test it with our code tomorrow and aim to get it merged then for you. Looks ok otherwise.

**Pull request merged**